src/options.js

```diff
@@ -48,7 +48,10 @@ export const preSpawnSchema = Joi.object({
           otherwise: path
         }).label('path')
     },
-    json: (schema) => Joi.object().allow('', null).required().label('json object'),
+    json: (schema) => Joi.alternatives().try(
```

Comment:
I'm not sure If I understand this change correctly. Isn't there any Joi.json instead?
I thought that `Joi.object()` would treat as JS Object so, same as `typeof [] === 'object'`

Reply:
Unfortunately that doesn't seem to be the case. If I send an array as input I get an error
**ValidationError: "json object" must be of type object**

You can checkout my test cases (99f8ae6b3383547d408f5e2aa282ab58f930a92a) to see what I'm trying to do.

Brief example:
```
const jq = require('node-jq')

obj = ["asdf","test"]

jq.run(".", JSON.stringify(obj), {'input': 'json'})
.then( out => {
    console.log("out:", out)
})
.catch( err => {
    console.error(err.message)
});
```

Reply:
Yep, totally get the why. I even wonder why do we even need Joi for the input tbh.

But this PR looks good.